Add_User.py

```diff
@@ -0,0 +1,13 @@
+def Add_User():
+    file = open("accountfiles.txt","a")
+    #Calculates ID using the length of the file
+    with open(r"accountfiles.txt", 'r') as fp:
+        count = 1
+        for line in enumerate(fp):
```

Comment:
Why enumerate? We are not using the count returning from this function but an external count defined by us